core/src/main/java/com/sequenceiq/cloudbreak/freeipa/FreeIPAAdapter.java

```diff
@@ -0,0 +1,20 @@
+package com.sequenceiq.cloudbreak.freeipa;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.FreeIpaV1Endpoint;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.describe.DescribeFreeIpaResponse;
+
+@Service
+public class FreeIPAAdapter {
```

Comment:
FreeIpa is called FreeIpa everywhere in the code,  so I think it is better to change FreeIPA to FreeIpa